go.mod

```diff
@@ -48,7 +48,7 @@ require (
 	github.com/dgraph-io/ristretto v0.1.1
 	github.com/djherbis/times v1.6.0
 	github.com/elastic/beats/v7 v7.0.0-alpha2.0.20240902050359-30320a3bfcfc
-	github.com/elastic/e2e-testing v1.99.2-0.20231005090720-556e60d449dc
+	github.com/elastic/e2e-testing v1.2.3
```

Comment:
Why did we bump this one too?

Reply:
`magefile.go` uses a single function from `e2e-testing`, and `k8s.io/kubernetes` is a transient dependency of `e2e-testing` as well.

```
go: github.com/elastic/e2e-testing@v1.99.2-0.20231005090720-556e60d449dc requires
        github.com/testcontainers/testcontainers-go@v0.13.0 requires
        github.com/containerd/containerd@v1.5.9 requires
        github.com/containerd/aufs@v1.0.0 requires
        github.com/containerd/containerd@v1.5.0-beta.3 requires
        github.com/Microsoft/hcsshim@v0.8.15 requires
        github.com/containerd/containerd@v1.5.0-beta.1 requires
        github.com/Microsoft/hcsshim/test@v0.0.0-20201218223536-d3e5debf77da requires
        github.com/Microsoft/hcsshim@v0.8.7 requires
        k8s.io/kubernetes@v1.13.0 requires
        k8s.io/api@v0.0.0: reading k8s.io/api/go.mod at revision v0.0.0: unknown revision v0.0.0
```